projects/plugins/social/readme.txt

```diff
@@ -73,14 +73,16 @@ You can connect to any of the following networks:
 * Instagram Business
 * Nextdoor
 * Mastodon
+* Threads
+* Bluesky
 
 After you add a new connection, you have the option to mark the connection as shared, meaning it can also be used by any other users on your site who have the ability to publish posts.
 
 To make the connection available to all users, check the box labeled "Mark the connection as shared".
 
 = To which social media platforms can I post using Jetpack Social? =
 
-You can post to Facebook, Instagram Business, Tumblr, Mastodon, Linkedin and Nextdoor. We are working hard to increase our social share integrations.
+You can post to Facebook, Bluesky, Threads, Instagram Business, Tumblr, Mastodon, Linkedin, and Nextdoor. We are working hard to increase our social share integrations.
```

Comment:
Is it correct to have a `,` before an `and`?

Reply:
Yay Oxford comma :)